cddl/raw-integral-type-choice.cddl

```diff
@@ -0,0 +1,5 @@
+raw-integral-type-choice /= integral / tagged-integral-range
+integral-range = [min: opt-integral, max: opt-integral]
+tagged-integral-range = #6.564(integral-range)
+integral = int / #6.2(bytes) / #6.3(bytes)
+opt-integral = integral / null
```

Comment:
What does “opt” stand for?  Optional?  If so, it sounds a bit odd.

What about a bit more expressively:

```
integer-range = [min: integer / negative-inf, max: integer / positive-inf]

negative-inf = null
positive-inf = null
```